gradle/libs.versions.toml

```diff
@@ -119,7 +119,7 @@ detektFormatting = { module = "io.gitlab.arturbosch.detekt:detekt-formatting", v
 # https://github.com/robolectric/robolectric/releases
 robolectric = "org.robolectric:robolectric:4.7.1"
 
-moshiKotlin = { module = "com.squareup.moshi:moshi-kotlin", version.ref = "moshi" }
```

Comment:
We don't need the `moshiKotlin`. It is needed only for reflection without code generation

Reply:
good find. i didn't noticed that
also, not the best artifact name ) should be something with reflection in it i think